### PR TITLE
[MONITOR-5545] Update page "Set up integrations to receive alerts"

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Renamed  page to "Triggering runs with webhooks"
- Moved page to "Running Collections" section in left nav
- Edited page and removed references to monitors
- Update redirects

### CODE REVIEW

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ In addition to using the Collection Runner in Postman, you can use collection ru
 
 * The Postman command-line interface [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) enables you to run collections and build them into your development pipeline, responding to test outcomes to maintain your API performance.
 * Adding a [monitor](/docs/monitoring-your-api/intro-monitors/) to your collection lets you schedule collection runs and stay informed of any issues.
-* You can also set up a [monitoring webhook](/docs/monitoring-your-api/integrations-for-alerts/) to trigger a collection at a specific time with your own custom payload.
+* You can also set up a [collection webhook](/docs/running-collections/collection-webhooks/) to trigger a collection run at a specific time with your own custom payload..
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/intro-to-collection-runs.md:114]: two periods at end

**Review by jkonrath-postman — APPROVED** [review]
One little thing, but otherwise LGTM

**Review by bpedro — APPROVED** [review]
LGTM. Thanks!

**Comment by giridharvc7** [comment]
LGTM 👍

**Review by giridharvc7 — APPROVED** [review]

**Pull request merged**